app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::TopicsController < ApplicationController
+  # con este skip me salteo el token, después se vuelve a activar?
+  skip_before_action :verify_authenticity_token
+  
+  def index
+    # head :ok
+    topics = Topic.all
+    render json: topics, status: 200
+  end
+
+  def show
+  end
+
+  def create
+    topic = Topic.new(
+      label: topic_params[:label],
+      icon: topic_params[:icon]
+    )
+    if topic.save
+      render json: topic, status: 200
+    else
+      render json: {
+        error: "Error Creating topic"
+      }
+    end
+  end
+
+  def update
+  end
+
+  def destroy
+  end
+
+  # this method controls the parameters we can pass by api to create a new topic
+  private
+  def topic_params
+    params.require(:topic).permit([
+      :label,
+      :icon
+    ])
```

Comment:
you can avoid the array here, just send the attributes you permit directly to the `permit` method:
```suggestion
    params.require(:topic).permit(
      :label,
      :icon
    )
```